src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,16 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    public static class ColorWheel {
+        public static final double[] RED = {0, 0, 0};
+        public static final double[] GREEN = {0, 0, 0};
+        public static final double[] BLUE = {0, 0, 0};
+        public static final double[] YELLOW = {0, 0, 0};
```

Comment:
add a comment to explain the values.
does it RGB (of course), the values is between 0-1 or 0-255?